src/ec/packages/ec-component-blockquote/blockquote.story.js

```diff
@@ -8,17 +9,49 @@ import data from '@ecl/ec-specs-blockquote/demo/data';
 import blockquote from './ecl-blockquote.html.twig';
 import notes from './README.md';
 
+// Labels for the group ids
+const optionalGroupId = 'Optional elements';
+const requiredGroupId = 'Mandatory elements';
+const useCasesGroup = 'Use cases';
+
+// Preserve the adapted specs.
+const defaultData = { ...data };
+
+// declare your toggled data
+const authorBtnToggler = () => {
+  defaultData.author = defaultData.author ? false : data.author;
+};
+
+const formatQuote = passeddata => {
```

Comment:
passeddata? :) something different, maybe..? 
I wonder the reason why you are checking for the passeddata.citation like it might not be in place.
In your logic the author might not be present, but as we saw also author is a mandatory field, so...